### PR TITLE
add sample webapp and resource in the same application

### PR DESCRIPTION
**Pull request by chenrujun**
Refs:  https://github.com/Azure-Samples/azure-spring-boot-samples/pull/5

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @backwind1233 , please help to review this PR.

File: aad/azure-spring-boot-sample-active-directory-webapp-and-resource-server/src/main/java/com/azure/spring/sample/aad/controller/webapi/WebApiController.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    private WebClient webClient;
+
+    @GetMapping("/hello")
+    public String hello() {
+        return "hello";
+    }
+
+    @GetMapping("/webapiC")
+    @PreAuthorize("hasAuthority('SCOPE_WebApiC.SampleScope')")
+    public String webapiD() {
```
Comment (backwind1233) [inline aad/azure-spring-boot-sample-active-directory-webapp-and-resource-server/src/main/java/com/azure/spring/sample/aad/controller/webapi/WebApiController.java:42]: webapiD？ why not webapiC

**Review by backwind1233 — APPROVED** [review]
LGTM

File: aad/azure-spring-boot-sample-active-directory-webapp-and-resource-server/src/main/java/com/azure/spring/sample/aad/controller/webapi/WebApiController.java (hunk 1 above)
Comment (chenrujun) [reply aad/azure-spring-boot-sample-active-directory-webapp-and-resource-server/src/main/java/com/azure/spring/sample/aad/controller/webapi/WebApiController.java]: Updated.

**Review by backwind1233 — APPROVED** [review]

**Pull request merged**